src/utils/routes.js

```diff
@@ -23,6 +23,7 @@ const userProfilePrefix = '/user-profile/';
 const documentInputPrefix = '/doc-inputs/';
 const mediaLibraryPrefix = '/media-library/';
 const samlAuthLoginPrefix = '/saml-auth/login/';
+const documentCategoriesPrefix = '/doc-categories/';
```

Comment:
In the controller I went with `document-categories`. I honestly would move away from `doc` and `rev` with new logic and stick with full names. I think we should have done the same with `doc-inputs`.
What are your thoughts?

Reply:
Changed it to `document-categories` as agreed.